src/services/aws/aws.ts

```diff
@@ -29,7 +34,7 @@ const awsBaseQuery = retry(
     mode: "cors",
     async prepareHeaders(headers) {
       if (headers.get("authorization") === TEMP_JWT) {
-        headers.set("authorization", `Bearer ${await jwtToken()}`);
+        headers.set("authorization", await jwtToken());
```

Comment:
Was this by mistake or was AWS updated to no longer expect `"Bearer"`?

Reply:
no need for `Bearer` per ser @jp-mariano